cli/ballerina-tool/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -7,21 +7,21 @@ SYNOPSIS
                   <balfile | module-name | executable-jar> [args...]
 
 DESCRIPTION
-    Run command executes the given program or module. This is a
-    convenient command which will build the program and execute it.
+    The `ballerina run` command executes the given Ballerina program or module. This is a
+    convenient command, which will build the program and execute it.
 
     If a Ballerina source file (.bal file) or a module is
-    given, the run command build and runs it. The compilation
-    is done internally for single bal files and executable will not be created.
+    given, the run command builds and runs it. The compilation
```

Comment:
Should we mention that it won't run the tests either?

Reply:
It need not be specifically mentioned.